sdks/python/container/boot.go

```diff
@@ -91,7 +92,12 @@ func main() {
 			"--container_executable=/opt/apache/beam/boot",
 		}
 		log.Printf("Starting worker pool %v: python %v", workerPoolId, strings.Join(args, " "))
-		if err := execx.Execute("python", args...); err != nil {
+		pythonVersion, err := expansionx.GetPythonVersion()
+		if err != nil {
+			log.Print("Python SDK worker pool exited.")
```

Comment:
Can we buble up the "no python installation found" error instead?
Should we use  `log.Fatalf` instead?

Reply:
yeah